distro/otel.go

```diff
@@ -58,11 +60,18 @@ func (s SDK) Shutdown(ctx context.Context) error {
 func Run(opts ...Option) (SDK, error) {
 	c := newConfig(opts...)
 
+	// Unify the SDK logging with OTel.
+	otel.SetErrorHandler(otel.ErrorHandlerFunc(func(e error) {
+		c.Logger.Error(e, "OpenTelemetry error")
+	}))
+	otel.SetLogger(c.Logger)
+
 	if c.Propagator != nil && c.Propagator != nonePropagator {
 		otel.SetTextMapPropagator(c.Propagator)
 	}
 
 	if c.TraceExporterFunc == nil {
+		c.Logger.V(1).Info("tracing disabled: no trace exporter")
```

Comment:
Can we provide a suggestion to users here, similar to what you've added in 93 to 94?

Reply:
This should only ever be reached if the user takes a positive action to set no trace exporter. Even if they make a mistake and set an invalid value the SDK will still use the default exporter. I'm open to suggestions of what we should recommend, but I anticipate users will get here because they want to disable tracing. This log message is only intended to confirm they did so.

Reply:
Got it. How about something along the lines of...

```suggestion
		c.Logger.V(1).Info("OTEL_TRACES_EXPORTER is empty or invalid: Tracing disabled")
```

(If that is indeed the var they'd use and the cases where the condition would be triggered.)

Reply:
You can get here by passing `OTEL_TRACES_EXPORTER=none` or passing `WithTracesExporter(nil)` directly as an option. If `OTEL_TRACES_EXPORTER` is invalid that is also logged as an error and the default OTLP is used instead.

Reply:
Also, probably worth mentioning for completeness, if `OTEL_TRACES_EXPORTER` is empty and no `WithTracesExporter` option is passed the default OTLP exporter is also used.

Reply:
Thanks! I've updated my comment to reflect that.